src/styles/variables/typography.scss

```diff
@@ -1,67 +1,70 @@
 // Auto-generated file by update:css-variables script. Do not edit manually
-html {--font-family-body: "Roboto", sans-serif;
---font-family-heading: "Roboto", sans-serif;
+html {
+  --font-family-body: "Roboto", sans-serif;
+  --font-family-heading: "Roboto", sans-serif;
 
-& [dir="rtl"] {
-      --font-family-body: "IBM Plex Sans Arabic", sans-serif;
-      --font-family-heading: "IBM Plex Sans Arabic", sans-serif;}
+  & [dir="rtl"] {
+    --font-family-body: "IBM Plex Sans Arabic", sans-serif;
+    --font-family-heading: "IBM Plex Sans Arabic", sans-serif;
+  }
 
---font-size-0: 0.75rem;
---font-size-1: 0.875rem;
---font-size-2: 1rem;
---font-size-3: 1.125rem;
---font-size-4: 1.25rem;
---font-size-5: 1.5rem;
---font-size-6: 1.75rem;
---font-size-7: 2rem;
+  --font-size-0: 0.75rem;
+  --font-size-1: 0.875rem;
+  --font-size-2: 1rem;
+  --font-size-3: 1.125rem;
+  --font-size-4: 1.25rem;
+  --font-size-5: 1.5rem;
+  --font-size-6: 1.75rem;
+  --font-size-7: 2rem;
 
---font-weight-default: 400;
---font-weight-thick: 500;
+  --font-weight-default: 400;
+  --font-weight-thick: 500;
 
---letter-spacing-medium: 0.02em;
---letter-spacing-none: 0;
---letter-spacing-small: 0.01em;
+  --letter-spacing-medium: 0.02em;
+  --letter-spacing-none: 0;
+  --letter-spacing-small: 0.01em;
 
```

Comment:
@ArkadiuszKrzymienski @sonya0504 can we remove these empty spaces?

Reply:
> Love IT! Pleas remove empty lines and ping me. We will merge this PR. Awesome!

@pspaczek ping ping ping! 🔔